app/scripts/controllers/account-tracker-controller.test.ts

```diff
@@ -185,7 +191,7 @@ function withController<ReturnValue>(
     ...accountTrackerOptions,
   });
 
-  return fn({
+  return await fn({
```

Comment:
I just followed how other controllers tests are done. But I believe in this particular case, returning await fn() is not mandatory because we aren’t wrapping the return inside a try-catch block.